### ISSUE TITLE
The sound track is not audible

### ISSUE THREAD

**Issue by shivakumars** [rborn/TitaniumYoutubePlayer#1]
I m using this in one of my application for iOS and playing youtube videos. The thing is, i can see only the video, i m not able hear the audio of it. 

I tried,
- useApplicationAudioSession : true
- volume : 1
  attributes on video player, but it didn't help. Do you have any idea to resolve it.

Note: I m using device (iPhone) for development. Not simulator.

**Comment by rborn** [comment]
I don't have any issue of this kind. You have to be sure the video can run on mobile.
If you use the youtube api when you search for videos set the format to 6
Something like this:

```
    var url = 'http://gdata.youtube.com/feeds/api/videos?q='+
    data.term+'&orderby=relevance&v=2&alt=json&format=6&max-results='+(data.limit||50)+'&restriction='+data.country+'&start-index='+((data.start||0)+1);
```

Can you see/hear the video in Safari mobile?

**Comment by shivakumars** [comment]
Ya, the same video is fine when played on safari and youtube app.

**Comment by rborn** [comment]
No idea, give me the link to the video is not working for you.

**Comment by shivakumars** [comment]
I guess the problem is different. Its not because of the link returned, i guess. The sound is audible when i use my headset. But it is not audible without headset. But the same video plays fine even without headset in other ways that i mentioned previously. Confused a little, but i will sort out. Thank you. :)

Edit: I did it. I made useApplicationAudioSession : false, which will create new audio session. That makes it to work. Thanks anyway.
